src/point_one/fusion_engine/messages/defs.h

```diff
@@ -39,7 +39,7 @@ enum class MessageType : uint16_t {
   POSE_AUX = 10003, ///< @ref PoseAuxMessage
   CALIBRATION_STATUS = 10004, ///< @ref CalibrationStatusMessage
   RELATIVE_ENU_POSITION = 10005, ///< @ref RelativeENUPositionMessage
-  RELATIVE_ENU_HEADING = 10006, ///< @ref RelativeENUHeadingMessage
+  HEADING_MEASUREMENT = 10006, ///< @ref HeadingMeasurement
```

Comment:
Let's move this into the sensor measurement block of values below, so change to 11001 here and in python

Reply:
(and move it down below IMU_MEASUREMENT)

Reply:
You also need to update the `to_string()` function below, including reording to match when you move this value below IMU

Reply:
ok doing now